packages/cli/src/config.js

```diff
@@ -51,7 +51,7 @@ module.exports.get = async (key, options = {}) => {
 
   if (!globalConfig && !localConfig) return
 
-  const mergedConfig = deepExtend(globalConfig, localConfig)
+  const mergedConfig = deepExtend(globalConfig || {}, localConfig || {})
```

Comment:
Why don't we handle the default case (`{}`) inside `getConfig`?

Reply:
I need to distinguish between undefined (no file present) or empty config. But probably I can add a default if none on `getConfig`

Reply:
Solved.